lib/member_page.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+require 'scraped'
+require_relative 'remove_session_from_url_decorator'
+require_relative 'parsed_date_string'
+require_relative 'strategies/live_request_with_cookie'
```

Comment:
it seems odd that there's a 'strategies/' directory, but not, say, a 'decorators/'. I don't really care whether everything is in separate directories or all in one, but consistency would be good.